.travis.yml

```diff
@@ -27,5 +27,5 @@ before_script:
   - android-wait-for-emulator
   - adb shell input keyevent 82 &
 script:
-  - export ANDROID_NDK_HOME=/usr/local/android-sdk/ndk-bundle
-  - bash ./gradlew build connectedCheck 
+  - export ANDROID_NDK_HOME=
```

Comment:
But unset won't export it to the calling environment.  I need to look into this.  It may be the gradle upgrade solves this without this additional line.